NitroxClient/GameLogic/Spawning/WorldEntities/PlaceholderGroupWorldEntitySpawner.cs

```diff
@@ -63,19 +69,31 @@ public IEnumerator SpawnAsync(WorldEntity entity, Optional<GameObject> parent, E
                     {
                         metadataProcessor.Value.ProcessMetadata(gameObject, placeholder.Metadata);
                     }
-                }
-                else
-                {
-                    Log.Debug($"Unhandled child type {child}");
-                    continue;
-                }
-            }
-        }
 
-        public bool SpawnsOwnChildren()
-        {
-            return true;
+                    break;
+                case WorldEntity slotEntity:
+                    IWorldEntitySpawner spawner = spawnerResolver.ResolveEntitySpawner(slotEntity);
+                    Optional<GameObject> slotEntityParent = Optional.Of(prefabPlaceholder.gameObject);
+
+                    TaskResult<Optional<GameObject>> slotEntityTaskResult = new();
+                    yield return spawner.SpawnAsync(slotEntity, slotEntityParent, cellRoot, slotEntityTaskResult);
+
+                    if (slotEntityTaskResult.value.HasValue)
+                    {
+                        // For some reasons it's not zero after spawning so we reset it here
+                        slotEntityTaskResult.value.Value.transform.localPosition = Vector3.zero;
```

Comment:
Is it possible that the local rotation isn't equal to the Identity?

Reply:
Need to check this with a side by side comparison to base SN.